### PR TITLE
Make summary retrieval command accept array and more

### PR DESCRIPTION
**Pull request by RicoVZ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Makes the `triage-get-sample-summary` accept an array of sample IDs. This is so the command can be used in a Genericpolling block. This requires the command to be an array. This PR also updates the docker image version and causes the `interactive` argument for the `triage-submit-sample` command to actually be used, instead of ignored.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by RicoVZ** [comment]
I set the state to 'in progress'. I just found out some more changes are required.

**Comment by RicoVZ** [comment]
It's ready now.

File: Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.py [hunk 1]
```diff
@@ -105,12 +119,21 @@ def get_sample(client: Client, **args) -> CommandResults:
 
 
 def get_sample_summary(client: Client, **args) -> CommandResults:
-    sample_id = args.get("sample_id")
-    r = client._http_request("GET", f"samples/{sample_id}/summary")
+    outputs = []
+    for sample_id in argToList(args.get("sample_id", "")):
```
Comment (yaakovpraisler) [inline Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.py:123]: Isn't there an option to make the API call one time for all IDs?

File: Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.yml [hunk 2]
```diff
@@ -109,11 +109,20 @@ script:
...
       required: true
       secret: false
+    - name: password
+      description: A password that may be used to decrypt the provided file, usually an archive
+      isArray: false
+    - name: timeout
+      description: The timeout in seconds of the behavioral analysis
+      isArray: false
+    - name: network
+      description: 'The type of network routing to use: internet, drop, tor, sim200, or sim404'
```
Comment (yaakovpraisler) [inline Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.yml:124]:
```suggestion
      description: The type of network routing to use: internet, drop, tor, sim200, or sim404
```

Comment (yaakovpraisler) [inline Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.yml:124]: If the options for this argument is only what you mentioned here in the description, then maybe you can do the argument as PRDEFINED
https://xsoar.pan.dev/docs/documentation/documentation_tips#common-command-arguments

File: Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.yml [hunk 3]
```diff
@@ -182,7 +191,7 @@ script:
   - arguments:
     - default: false
       description: Sample's unique identifier, can be found using the query samples command
```
Comment (yaakovpraisler) [inline Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.yml:193]: Please write here in the description that the input can be multiple IDs as comma separated

**Review by yaakovpraisler — CHANGES_REQUESTED** [review]
Hi @RicoVZ,
Thank you for your contribution! Nice work :)
Please see my comments -

File: Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.py (hunk 1 above)
Comment (RicoVZ) [reply Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.py:123]: There isn't such an API, no. Each ID is a separate analysis.

**Review by yaakovpraisler — APPROVED** [review]

**Comment by yaakovpraisler** [comment]
@RicoVZ - Please update your branch from master so I can merge.

**Comment by RicoVZ** [comment]
Please ignore this PR. See new one: https://github.com/demisto/content/pull/23684

**Pull request closed without merging**